src/Components/ShareImageModal/RegionsSelect.tsx

```diff
@@ -82,7 +82,7 @@ const RegionsSelect = ({
   });
 
   if (!isSuccess) {
-    return undefined;
```

Comment:
Returning `undefined` should actually be allowed now, after e5896d8d9c56713a0a3668e80f44547830812175. You might need to update your dependencies with an `npm ci` to get your tsserver to not complain about it.